src/main/java/org/gridsuite/study/server/dto/impacts/CollectionElementImpact.java

```diff
@@ -15,12 +15,14 @@
  *
  * @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
  */
-@Setter
-@Getter
 @SuperBuilder
+@Data
+// @AllArgsConstructor // TODO activate if one member is added
 @NoArgsConstructor
```

Comment:
This one is useless too here, because would be created anyway.

Reply:
@Data contains @RequiredArgsConstructor not @NoArgsConstructor. If I remove it, some unit tests fail.
Please suggest a better solution.

Reply:
In this case @requiredargsconstructor is equivalent to  @NoArgsConstructor. I tried to remove it and all the tests passed.